src/deriver/queue.py

```
@@ -114,7 +115,8 @@
     async def get_available_sessions(self, db: AsyncSession):
         """Get available sessions that aren't being processed"""
         # Clean up stale sessions
-        five_minutes_ago = datetime.utcnow() - timedelta(minutes=5)
+        stale_delta = timedelta(minutes=settings.DERIVER.STALE_SESSION_TIMEOUT_MINUTES)
+        five_minutes_ago = datetime.utcnow() - stale_delta
```

Ruff (0.11.9)

`datetime.datetime.utcnow()` used

(DTZ003)